test/known_issues/test-fs-fchmod-12835.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+if (common.isWindows) {
```

Comment:
This is not necessary if `known_issues.status` is properly updated, of I get this right.